dumper/lib-dumper-spi/src/main/java/com/google/edwmigration/dumper/plugin/lib/dumper/spi/TeradataLogsDumpFormat.java

```diff
@@ -180,6 +180,7 @@ enum HeaderForAssessment {
     }
 
     enum HeaderLSql {
+        QueryID,
```

Comment:
The connector uses this to drive the underlying select. Do we know why this got out of sync? Is someone trying to read this column, which the connector isn't currently writing?

Reply:
This field is used by [`TeradataLogDumpCsvTranspilerIngester`](https://source.corp.google.com/piper///depot/google3/java/com/google/cloud/helix/edwmigration/translation/engine/lang/teradata/transpiler/TeradataLogDumpCsvTranspilerIngester.java;rcl=478391678;l=90) and I presume that isn't recent; [`TeradataLogsConnector`](https://github.com/google/dwh-migration-tools/blob/849e0849f5e65eb78141f19d756f88723813ccf4/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java#L98) writes this field.